components/css/buckyless/notifications.less

```diff
@@ -56,6 +56,7 @@
         .md-list-item-text {
           padding: 16px 0;
 
+          /* Priority notifications are deprecated. */
```

Comment:
It looks like these deprecation comments in other files are uppercase. Should this be as well?

Reply:
Good call on consistency. Normalized to "DEPRECATED".